azure-cron.yml

```diff
@@ -57,17 +57,21 @@ jobs:
       - checkout: self
       - bash: ci/dev-env-install.sh
         displayName: 'Build/Install the Developer Environment'
+      - template: ci/bash-lib.yml
+        parameters:
+          var_name: bash-lib
       - bash: |
           set -euo pipefail
           eval "$(dev-env/bin/dade assist)"
 
           bazel build //ci/cron:cron
+
+          source $(bash-lib)
           # 90 minutes should provide enough overlap for an hourly
           # cronjob.
-          ./bazel-bin/ci/cron/cron bazel-cache --age 90 --delete
+          gcs ./bazel-bin/ci/cron/cron bazel-cache --age 90 --delete
```

Comment:
As it stands this would end up running `gsutil ./bazel-bin/ci/cron bazel-cache --age 90 --delete`, but I like the idea. Perhaps it's worth changing `bash-lib` so that `gcs` really is the `with-gcs-credentials` type of wrapper you're going for here.

Reply:
Yeah I just realized that, I copied the logic for now instead of calling gcs. Do you want to align them in this PR or separately?

Reply:
Separately is fine, I've made a note of it. I don't want to delay this.